codefresh.yml

```diff
@@ -0,0 +1,39 @@
+# More examples of Codefresh YAML can be found at
+# https://codefresh.io/docs/docs/yaml-examples/examples/
+
+version: "1.0"
+# Stages can help you organize your steps in stages
+stages:
+  - "clone"
+  - "build"
+  - "test"
+
+steps:
+  clone:
+    title: "Cloning repository"
+    type: "git-clone"
+    repo: "nobl9/terraform-provider-nobl9"
+    # CF_BRANCH value is auto set when pipeline is triggered
+    # Learn more at codefresh.io/docs/docs/codefresh-yaml/variables/
+    revision: "${{CF_BRANCH}}"
+    git: "github"
+    stage: "clone"
+
+  test:
+    title: "Running test"
+    type: "freestyle" # Run any command
+    image: "hashicorp/terraform:1.0.11" # The image in which command will be executed
+    working_directory: "${{clone}}" # Running command where code cloned
+    commands:
+      - "apk add go make"
+      - "mkdir -p ~/.ssh"
+      - "echo ${{GH_SSH_PUBLIC}} > ~/.ssh/id_ed25519.pub"
+      - "echo ${{GH_SSH_PRIVATE}} | base64 -d > /root/.ssh/id_ed25519"
+      - "chmod 644 ~/.ssh/id_ed25519.pub"
+      - "chmod 600 ~/.ssh/id_ed25519"
+      - "eval  $(ssh-agent -s)"
+      - "ssh-add ~/.ssh/id_ed25519"
+      - "ssh -T -oStrictHostKeyChecking=accept-new git@github.com || true"
+      - 'git config --global --add url."git@github.com:".insteadOf "https://github.com/"'
+      - "make testacc"
```

Comment:
How do you provide configuration for the tests? running just `make testacc` will fail because we are missing:

```
NOBL9_URL=http://xxx/api
NOBL9_ORG=nobl9-dev
NOBL9_PROJECT=terraform
NOBL9_CLIENT_ID=xxx
NOBL9_CLIENT_SECRET=xxx
```